test/Validation.Tests/RequiresTests.cs

```diff
@@ -228,4 +230,22 @@ public void Defined_Int64Enum()
         InvalidEnumArgumentException ex = Assert.Throws<InvalidEnumArgumentException>(() => Requires.Defined((BigEnum)0x100000000, "parameterName"));
         this.logger.WriteLine(ex.Message);
     }
+
+    [Fact]
+    public void ValidElements()
+    {
```

Comment:
Nice set of tests. Can you add one for a `null` first argument?

Reply:
Perhaps another whose predicate *throws*.

Reply:
Yes, I will add those.